pkg/cdi/default-cache_test.go

```diff
@@ -115,7 +115,7 @@ devices:
 					filepath.Join(dir, "run"),
 				),
 			}
-			Configure(opts...)
+			_ = Configure(opts...) // always nil now, should we update and break API ?
```

Comment:
Would it make sense to check the error and fail a test if it's not nil? If/when Configure starts to return an error, we don't need to modify the tests.